### ISSUE TITLE
Feature request: option to show schemas as tree rather than picker

### ISSUE THREAD

**Issue by maxrothman** [TablePlus/TablePlus#2297]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): Postgres 11.5

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 354

Working with databases that make heavy use of schemas in TablePlus is burdensome. When switching between tables in different schemas, you constantly have to switch which schema you have selected in the schema picker in the bottom right. Other apps such as Postico display schemas as a tree, which makes it much easier to switch between these schemas:

<img width="189" alt="Screen Shot 2021-02-03 at 12 32 02 PM" src="https://user-images.githubusercontent.com/70533084/106785841-de05bc00-661b-11eb-8c99-f7a1bab831e7.png">

I know that typical users work in a single schema, but having the option to switch to a view better optimized for multiple schemas would be a huge improvement for the users that need it. Personally, I keep switching back and forth between TablePlus and Postico, and the only thing keeping me on Postico is this particular issue.

**Comment by qwesda** [comment]
This keeps me from switching as well.

**Comment by tjmcewan** [comment]
`Open Anything` is a good* navigation alternative once you're fully across the schema/table structure, but nothing beats having the visual layout of all tables in one view.

\* it would be a **_great_** alternative if TP had the option to preload tables from all schemas on startup.  As it stands you have to manually visit each schema first, _then_ you can properly use it for table nav.

**Comment by huyphams** [comment]
you can enable preload all schema item @tjmcewan on the left sidebar settings

**Comment by tjmcewan** [comment]
Aah, game changer!  I was searching through the Settings for ages.  Don't think I've noticed that icon before, took me a bit to figure out what you meant.  Thanks!  (Still a vote for the tree view tho) 🙏

**Comment by huyphams** [comment]
Yeah, I'm making an option where the user can choose the style of the left sidebar tree view vs current view.

**Comment by borispetrovdev** [comment]
> Yeah, I'm making an option where the user can choose the style of the left sidebar tree view vs current view.

Has this option been added yet?

**Comment by huyphams** [comment]
Not yet but we will support it @borispetrovdev  (next month 😅)
